### DISCUSSION TITLE
Assistance Needed for Account Recovery Dear Github Support Team

### DISCUSSION THREAD

**Discussion by Aquilaafuadajo** [community/community#133508] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

 A friend of mine is currently unable to receive my sign-in verification code via SMS and do not have access to recovery codes. Note: This person has initiated the 2FA account recovery process and submitted my SSH key three days ago, but has not yet received a response. This matter is urgent, and I would greatly appreciate your prompt assistance in resolving it. Thank you for your attention to this issue. Best regards

Ticket ID: 2900383

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Aquilaafuadajo 

Thank you and your friend for your patience. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). P The community is unable to assist with nor escalate your friend's ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note: the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects._
